viewing_party/party.py

```diff
@@ -1,23 +1,139 @@
 # ------------- WAVE 1 --------------------
 
+from shutil import move
+from tests.test_constants import FANTASY_1, FANTASY_2, GENRE_1, HORROR_1, RATING_1
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {"title": title, "genre": genre, "rating": rating}
+    if title and genre and rating:
+        return new_movie
+    else:
+        print(f'{new_movie} is None')
+
+# test 5
+
+
+def add_to_watched(user_data, movie):
+    user_data = {"watched": []}
+    user_data['watched'] = [movie]
+    return user_data

```

Comment:
delete line 18 and only keep line 19? since line 19 only adds new value to `user_data`

Reply:
Keeping line 19 will help us preserve info in `user_data` outside of the `watched` list, but we still lose anything else in `watched` since we assign `user_data['watched']` to a new list containing `movie`. We would need to change this line so that we append `movie` to the existing list:
```python
user_data['watched'].append(movie)
```

Reply:
I see... that makes sense. I really need to utilize .append() more :-) Thank you!